common/prettify.pm

```diff
@@ -670,6 +671,28 @@ sub Description ($)
 
 sub Timestamp ($)
 {
+    my $self = shift;
+    my $ts = shift;
```

Comment:
Does `$self` not count towards the sub's prototype?

Reply:
Yes and no?
https://perldoc.perl.org/perlsub#Prototypes
> Method calls are not influenced by prototypes either, because the function to be called is indeterminate at compile time, since the exact code called depends on inheritance.